src/vivarium_gates_nutrition_optimization_child/model_specifications/branches/scenarios.yaml

```diff
@@ -1,7 +1,7 @@
-input_draw_count: 5
+input_draw_count: 20
 random_seed_count: 20
 
-input_draws: [457, 169, 323, 392, 346] #, 480, 258, 446, 356, 273]
+input_draws: [457, 169, 323, 392, 346, 480, 258, 446, 356, 273, 103, 113, 137, 158, 177, 197, 21, 222, 253, 425]
```

Comment:
@rmudambi Im going to make a Jira ticket to get the draws and seeds so we don't have to specify it like this every time. This just is begging for us to make an error.